.github/workflows/e2e-test.yml

```diff
@@ -22,15 +22,15 @@ jobs:
 
       - name: Initiate Dockware
         run: |
-          docker pull dockware/dev:6.4.20.1
+          docker pull dockware/dev:latest
           docker run --rm -p 443:443 --name shopware6 \
             --mount type=bind,source="$(pwd)",target=/data/extensions/workdir \
-            --env PHP_VERSION=7.4 -d dockware/dev:6.4.20.1
+            --env PHP_VERSION=8.2 -d dockware/dev:latest
           sleep 30
           docker logs shopware6
-          docker exec shopware6 bash -c "mysql -u root -proot shopware -e \"UPDATE sales_channel_domain SET url='https://local.shopware.shop' WHERE url NOT LIKE 'default.%';\""
+          docker exec shopware6 bash -c "sudo mysql -u root -proot shopware -e \"UPDATE sales_channel_domain SET url='https://local.shopware.shop' WHERE url NOT LIKE 'default.%';\""
```

Comment:
why now we need sudo and before we didn't?

Reply:
New docker image for 6.5 doesn't allow GH action user to connect to mysql